Assignment

```diff
@@ -1 +1,83 @@
+import csv
 
+def read_weather_data(weather_data):
+    weather_data = []
+    with open( r"C:\Users\ue\Desktop\weatherAUS\weatherAUS.csv") as csvfile:
```

Comment:
@hareem211 Why you have used with operator?

Reply:
with operator is used when dealing with file...I've used with operator because it completely closed the file after its suite finishes